CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing to the Conjur Katacoda Scenarios
+
+For general contribution and community guidelines, please see the [community repo](https://github.com/cyberark/community).
+
+## Table of Contents
+- [Prerequisites](#prerequisites)
+- [Katacoda Documentation](#katacoda-documentation)
+- [Building and Testing](#building-and-testing)
+- [Troubleshooting Steps](#troubleshooting-steps)
+- [Contribution Workflow](#contribution-workflow)
+
+## Prerequisites
+The Katacoda CLI is recommended in order to create a new scenario. You can install the Katacoda CLI by running `npm install katacoda-cli --global` in the terminal.
+
+## Katacoda Documentation
+A link to the Katacoda documentation can be found [here](https://www.katacoda.community/welcome.html).
+
+Additionally, it is recommended that you look at the other scenarios in the `cyberark/katacoda-scenarios` to see how Katacoda has been used in other tutorials.
```

Comment:
This isn't a full http link

Reply:
Are you referring to the Katacoda documentation link or the repository path?